### ISSUE TITLE
Support for stapling information

### ISSUE THREAD

**Issue by cre8** [openid/OpenID4VP#185]
Some revocation protocols support stapling like OSCP and [Bitstring](https://www.w3.org/TR/vc-bitstring-status-list/#bitstringstatuslistcredential) to increase the privacy of the verifier that does not need to request the issuer for the information.

So my question is how should the holder pass the information to the verifier? In case of a verifiable presentation based on W3C-VC-DM, should it include the Bitstring-VC together in the presentation. How to do this in cases like CRL or OCSP where the object is not JSON based?

**Comment by peppelinux** [comment]
@cre8 did you see the OAuth Status Assertions? Here
https://peppelinux.github.io/draft-demarco-oauth-status-attestations/draft-demarco-oauth-status-attestations.html

**Comment by cre8** [comment]
@peppelinux I saw this, but this forced the issuer/status provider to use this mechanism instead of using the CRL approach.

**Comment by Sakurann** [comment]
How OSCP can be used for credential status management has not been standardized, so that should happen first before we can define anything in the protocol.

On the bit string based status management mechanism, it is part of the credential format, so it is out of scope for the protocol.
